src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -265,16 +265,20 @@
     <value>File Conflict</value>
   </data>
   <data name="Button_No" xml:space="preserve">
-    <value>No</value>
+    <value>_No</value>
+    <comment>Button_Yes, Button_No, ButtonYesToAll, Button_NoToAll should have underscore in different letters. See https://docs.microsoft.com/en-us/windows/uwp/design/input/access-keys#choose-access-keys</comment>
```

Comment:
I'd suggest checking with localization team to make sure they understand this, if you haven't already.

Reply:
Will follow up with them once this PR is merged.